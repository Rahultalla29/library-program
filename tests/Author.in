AUTHOR Jeff Hopkins
ADD COLLECTION sample1.csv
AUTHOR Harper Lee
AUTHOR J.K. Rowling
AUTHOR Steve Rogers
AUTHOR Douglas Adams
EXIT
