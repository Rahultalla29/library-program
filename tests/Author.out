user: No books in system.

user: 17 books successfully added.

user: To Kill a Mockingbird (Harper Lee)

user: Harry Potter 1 (J.K. Rowling)
Harry Potter 2 (J.K. Rowling)
Harry Potter 3 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)
Harry Potter 5 (J.K. Rowling)
Harry Potter 6 (J.K. Rowling)

user: No books by Steve Rogers.

user: The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)

user: Ending Library process.
